### DISCUSSION TITLE
How to deploy Adonisjs 5 on Digital ocean

### DISCUSSION THREAD

**Discussion by fenibofubara** [adonisjs/core#4019] [Help]
Hi guys, am having .env issues while trying to deploy to Digitalocean. Pls kindly help

https://stackoverflow.com/questions/74930254/how-can-i-deploy-adonis-js-5-on-digital-ocean

**Comment by e-e** [comment]
and you definitely have a `.env` file in the root of the project? it's not `.env.example` or something like that?

**Comment by LeCoupa** [comment]
You have to copy your .env file in your build folder

 npm run build
 cp .env build/.env

https://dev.to/lecoupa/deploy-adonis-5-in-production-with-ci-cd-44ch
